Slack.Automation/Promact.Erp.Core/ActionFilters/ApiExceptionLoggerFilter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Autofac.Extras.NLog;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http.Filters;
+
+namespace Promact.Erp.Core.ActionFilters
+{
+    public class ApiExceptionLoggerFilter : IExceptionFilter
+    {
+        private readonly ILogger _logger;
+
+        public ApiExceptionLoggerFilter(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public bool AllowMultiple { get { return true; } }
+
+        public Task ExecuteExceptionFilterAsync(
+                HttpActionExecutedContext actionExecutedContext,
+                CancellationToken cancellationToken)
+        {
+            return Task.Factory.StartNew(() =>
```

Comment:
Removed the create task, now only return Task.FromResult(0) to display this task executed successfully.